com.rq3.Reaction.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.rq3.Reaction
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Reaction
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.Reaction
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/sdl12-compat.json
+
+  - name: reaction
+    buildsystem: simple
+    build-commands:
+      - make DEFAULT_BASEDIR=${FLATPAK_DEST}/share/Reaction -j${FLATPAK_BUILDER_N_JOBS}
+      - mkdir -p ${FLATPAK_DEST}/bin/
+      - mkdir -p ${FLATPAK_DEST}/share/Reaction/
+      - make copyfiles COPYBINDIR=${FLATPAK_DEST}/bin/ COPYDIR=${FLATPAK_DEST}/share/Reaction/ NO_STRIP=1
+      - mv ${FLATPAK_DEST}/bin/Reactionded.* ${FLATPAK_DEST}/bin/Reactionded
+      - mv ${FLATPAK_DEST}/bin/Reaction.* ${FLATPAK_DEST}/bin/Reaction
+      - mv ${FLATPAK_DEST}/bin/*.so ${FLATPAK_DEST}/share/Reaction/
+      - install -Dm644 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 reaction.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/ReactionQuake3/reaction.git
+        tag: d214682124851c5fb4a1f7f254305ccfcec7bffc
+      - type: patch
+        path: resync.patch
```

Comment:
a 5000+ lines patch, that's a bit excessive and divergent from upstream.

Reply:
The necessary shader compilation crash fix is sadly included all in one commit which completes an ioQuake3 engine update. Both the 2012 release and the latest included commit are broken. Believe me, I tried a lot until I got this working.